LHM/Problems/beakjoon/10818.py

```diff
@@ -0,0 +1,20 @@
+#https://www.acmicpc.net/problem/10818
+N = int(input())
+print(N)
+input_val =[]
+
+
+for i in range(0,N):
+    input_val.append(input())
+
+max = input_val[0]
+min = input_val[0]    
+
+for i in range(0,N):
+    if int(input_val[i]) > int(max):
+        max = input_val[i]
+    
+    if int(input_val[i]) < int(min):
+        min = input_val[i]
```

Comment:
이렇게 간략하게 사용하는 것도 가능한 방법일거 같습니다.

for에 대한 사용 방법이 C와 다르게 input_val 안에 있는 정보를 다 사용하는 동안으로 바로 읽어올 수 있습니다.
(index에 맞춰 불러오지 않고, 간략하게 사용가능함.)

```suggestion
for val in input_val:
    if val > max:
        max = val
    if val < min:
        min = val
```

Reply:
문제에 "서로 다른 N개"라는 표현이 없으니 Min, Max 비교 시 등호(=)도 고려 필요해보임